main.js

```diff
@@ -686,6 +686,35 @@ function _select_Gateway() { // Check for Gateway used to
     }
 
     /*
+     * Helper function to add "Mass Gathering" profile
+     */
+    function addMassProfile(prof, T1MassGat, T1MassRef, T1Gat, T1Ref, T2MassGat, T2MassRef, T2Gat, T2Ref, T3MassGat, T3MassRef, T3Gat, T3Ref, T4MassGat, T4MassRef, T4Gat, T4Ref)

```

Comment:
Sorry but I'm not merging this. These kind of *useless* functions and moreover with a ton of useless args...

Either set up the the profiles like the other ones we already have or don't. Having such a mess with random functions here and there (moreover such **bad** functions) is an incredibly huge pain for maintainability in the future for whoever picks up the code.